Modules/Loadable/Tables/qSlicerTablesReader.cxx

```diff
@@ -114,6 +119,30 @@ bool qSlicerTablesReader::load(const IOProperties& properties)
     name = properties["name"].toString();
     }
   std::string uname = this->mrmlScene()->GetUniqueNameByString(name.toLatin1());
+
+  if (fileName.contains(".db"))
+    {
+   uname = "";
+    std::string dbname = std::string("sqlite://") + fileName.toStdString();
+    vtkSQLiteDatabase *database = vtkSQLiteDatabase::SafeDownCast( vtkSQLiteDatabase::CreateFromURL(dbname.c_str()));
+    if (!database->Open("", vtkSQLiteDatabase::USE_EXISTING))
+      {
+      bool ok;
+      QString text = QInputDialog::getText(0, tr("QInputDialog::getText()"),
+                                           tr("Database Password:"), QLineEdit::Normal,
+                                           "", &ok);
+      if (ok && !text.isEmpty())
+        {
+        uname = text.toStdString();
```

Comment:
use toLatin1 for consistency (used most places in Slicer source code instead of toStdString)